### ISSUE TITLE
ruby_deep_clone 0.7.1 on rubygems does not contain header file

### ISSUE THREAD

**Issue by NickDryslewski** [gmodarelli/ruby-deepclone#11]
The currently released gem for 0.7.1 on rubygems.org (https://rubygems.org/gems/ruby_deep_clone/) does not include the header file (ext/deep_clone/deep_clone.h) needed to compile the gem. When compiling, you get:

```
gem install ruby_deep_clone -v '0.7.1'
Building native extensions.  This could take a while...
ERROR:  Error installing ruby_deep_clone:
ERROR: Failed to build gem native extension.

current directory: /Users/name/.rvm/gems/ruby-1.9.3-p551/gems/ruby_deep_clone-0.7.1/ext/deep_clone
/Users/name/.rvm/rubies/ruby-1.9.3-p551/bin/ruby -r ./siteconf20160420-49369-1uvptbx.rb extconf.rb
creating Makefile

current directory: /Users/name/.rvm/gems/ruby-1.9.3-p551/gems/ruby_deep_clone-0.7.1/ext/deep_clone
make  clean

current directory: /Users/name/.rvm/gems/ruby-1.9.3-p551/gems/ruby_deep_clone-0.7.1/ext/deep_clone
make
compiling deep_clone.c
deep_clone.c:1:24: fatal error: deep_clone.h: No such file or directory
#include "deep_clone.h"
                    ^
compilation terminated.
make: *** [deep_clone.o] Error 1

make failed, exit code 2
```

This was fixed in the commit 1c154c46d90bed3260df075eb553c22fa630515d. Can you please reupload the gem to rubygems?

**Comment by balmma** [comment]
Strange enough the header files where removed in commit 11d2b95. Put it back and just released 0.7.2.

**Comment by NickDryslewski** [comment]
Thanks!
